src/store/balances/balances-selectors.ts

```diff
@@ -0,0 +1,4 @@
+import {useSelector} from '../index.ts';
+
+export const useBalancesListSelector = () =>
```

Comment:
create PR into the `main` branch.  Cuz I see redundant changes in this PR.  
Do you use this state? Don't see ussages of `useBalancesListSelector`